equinix-openapi-metal/src/main/java/com/equinix/openapi/ApiException.java

```diff
@@ -16,7 +16,6 @@
 import java.util.Map;
 import java.util.List;
 
-import javax.ws.rs.core.GenericType;
```

Comment:
A bunch of the generated changes are like this, removing an `import javax.ws.rs.core.GenericType` that doesn't appear to have been used in most files anyway.